experiments/dataSources/airtable/README.md

```diff
@@ -0,0 +1,104 @@
+
+## Create a Skuid data source
+
+In your Skuid site, configure a data source with these properties:
+
+- **Name**: Airtable
+- **Data Source Type**: REST
+- **URL**: ``https://api.airtable.com/v0/``
+- **Authentication Method**: No Authentication
+- **Use (Apex) Proxy**: Checked
+
+And within *Headers to send with every request*, append the following values in the **Common Request Parameters**:
+
+- Accept: ``application/json``
+- Content-Type: ``application/json``
+- Authorization: ``Bearer <paste your API key here>``
+
+If prompted for a Remote Site Setting, accept it. Otherwise, you'll need to create one manually. 
+
+## Configure a Skuid model
+
+When using Airtable as a data source, you can think of your sheets as *objects*. So for any sheets whose columns you wish to access, you'll need to create a model. Let's do that now.
+
+Go into an existing Skuid page or create a new one. Next, create a new model and configure its basic properties:
+
+- **Name**: Enter a name related to the model's purpose, perhaps related to the Airtable sheet's name.
+- **Data Source Type**: REST
+- **Data Source**: The Airtable data source you created above
+- **Id Field / Primary Key**: Id
+- **Model Behavior**: Read-Only
+
+We're not finished with this model's properties, but let's explain what else is happening here. With the model behavior set as read-only, you can enter your data source URL on this tab. Your method will have a data source URL that follows this basic format:
+
+``<baseId>/<sheetName>``
+
+With an actual base ID and sheet name (), it'll look something like this:
+
+``appnibfIletterstRN/Applicants``
+
+When combined with the API URL we set in the Skuid data source, Skuid will send requests to URLs that look like this:
+
+``https://api.airtable.com/v0/appnibfIletterstRN/Applicants``
+
+Notice the connection between the API documentation and how Skuid parses things? So let's set the rest of the properties here:
+
+- **Data Source URL**: ``<baseId>/<ObjectName>``
+- **Data Source HTTP Verb**: ``GET``
+- **Path to Contents**:  ``records``
+
+Airtable nests its records in a ``records`` key within its response, hence why we must set our path to contents to gain access to the fields we need.
+
+### Add fields
+
+**To repeat the warning from above**: Ensure that the sheet you're accessing within your Airtable base **does not have any empty rows**. Skuid will parse the metadata of your sheet based on the least populated row. Have an empty row? You won't be able to retrieve any fields. Only filled out one column in a row? You'll only be able to retrieve that one column. Make sure that your data is well-populated.
```

Comment:
```suggestion
**To repeat the warning from above**: Ensure that the sheet you're accessing within your Airtable base **does not have any empty rows**. Skuid will parse the metadata of your sheet based on the last populated row. Have an empty row? You won't be able to retrieve any fields. Only filled out one column in a row? You'll only be able to retrieve that one column. Make sure that your data is well-populated.
```

Reply:
I need to double check this, but this wasn't a typo. The row with the least information is what determined which fields Skuid made available to me. Gonna triple check that it wasn't just the last row though.

Reply:
Aaaaaand now I can't replicate this behavior in either fashion:
- Having an empty row in the middle does not affect the metadata.
- Having an empty row at the end does not affect the metadata.

@Rnhatch did see this behavior, but now I'm unsure what's going on. I'm going to revise this note to say "Having trouble with your metadata? Make sure your rows are populated" instead of having it be a definitive warning.

Reply:
How's this? And I'll remove the warning at the top of the doc.
```suggestion
**Note**: If you're having issues with Skuid retrieving metadata, try to avoid having *empty rows*. It seems that rows without each column filled out can sometimes cause issues with metadata retrieval. 
```

Reply:
@itmightbecody  If you want to commit your proposed change, I'll review this PR and we can get it out into the world.